doc/manual/manual.tex

```diff
@@ -5023,6 +5023,36 @@ \subsubsection{Prescribed internal velocity constraints}
     \label{fig:prescribed-velocity}
 \end{figure}
 
+\subsubsection{Artificial viscosity smoothing}
+\label{sec:artificial-viscosity-smoothing}
+\textit{This section was contributed by Ryan Grove}
+
+Standard finite element discretizations of advection-diffusion equations introduce unphysical oscillations around steep gradients. Therefore, stabilization must be added to the discrete formulation to obtain correct solutions. In ASPECT, we use the Entropy Viscosity scheme developed by Guermond et al.~in the paper \cite{guer11}. In this scheme, an artificial viscosity is calculated on every cell and used to try to combat these oscillations that cause unwanted overshoot and undershoot.  More information about how \aspect{} does this is located at \url{https://dealii.org/developer/doxygen/deal.II/step_31.html}.  
+
+Instead of just looking at an individual cell's artificial viscosity, improvements in the minimizing of the oscillations can be made by smoothing.  The motivation behind smoothing came from the idea of not only using a particular cell's artificial viscosity, but the artificial viscosity of neighboring cells as well.  The smoothing process refers to the comparing of the artificial viscosity of each cell with the artificial viscosity of each of the particular cell's neighboring cells and the maximum of this comparison is then used for future calculations.
```

Comment:
something like "maximum artificial viscosity taken over the neighboring cells, i.e.,
\[
  \bar{v_h}(T) = \max_{K \in \N(T)} v_h(K)
\]
where N(T) is the set containing T and the neighbors across the faces of T.
"